src/account-info/ProfileCard.js

```diff
@@ -57,6 +58,7 @@ class LogoutButton extends PureComponent<{| +dispatch: Dispatch |}> {
   onPress = () => {
     const { dispatch } = this.props;
     dispatch(tryStopNotifications());
+    dispatch(resetToAccountPicker());
     dispatch(logout());
```

Comment:
It's kind of awkward that we end up having to repeat these separately at each call site. It seems like when logging out, we're consistently always going to want to go to the account picker. (Or maybe an improvement: go conditionally to either the account picker if you have another account to use, or straight to the login flow (RealmScreen) if you don't! That would mean additional logic, which we certainly wouldn't want to repeat at each call site.)

The above comment about `loginSuccess` highlights this -- it looks like a couple of call sites there might have been missed.

One thing that would probably help is to wrap these two lines up in a thunk action creator. Then `accountActions` might not even export the plain-action creators like `logout`; all call sites would call the wrapper, and that thunk would dispatch the plain action as well as cause a navigation.